xain_fl/coordinator/round.py

```diff
@@ -22,10 +22,30 @@ def __init__(self, participant_ids: List[str]) -> None:
         self.participant_ids = participant_ids
         self.updates: Dict[str, Dict] = {}
 
+    def add_selected(self, more_ids: List[str]) -> None:
+        """Add to the collection of selected participants.
+
+        Args:
+            more_ids: ids of participants to add.
+        """
+
+        self.participant_ids.extend(more_ids)
+
+    def remove_selected(self, participant_id: str) -> None:
+        """Remove from the collection of selected participants.
+
+        Args:
+            participant_id: id of participant to remove.
+        """
+
+        self.participant_ids = [
+            id for id in self.participant_ids if id != participant_id
+        ]
```

Comment:
list comprehension for identity comprehension is an anti pattern which shouldn't be used. i would suggest to do the following:

```
try:
    self.participant_ids.remove(participant_id)
except ValueError:
    pass
```